cli/pkg/credentialmanager/handler_linux.go

```diff
@@ -62,7 +62,11 @@ func (cm *CredentialManager) GetCreds(namespace string) (url.URL, string, error)
 	// Check if creds file is specified in the 'KEPTNCONFIG' environment variable
 	if customCredsLocation, ok := os.LookupEnv("KEPTNCONFIG"); ok {
 		if customCredsLocation != "" {
-			return handleCustomCreds(customCredsLocation)
+			endPoint, apiToken, err := handleCustomCreds(customCredsLocation, namespace)
+			// If credential is not found in KEPTNCONFIG, use fallback credential manager
+			if apiToken != "" || err != nil {
```

Comment:
If there is some error fetching creds from the KEPTNCONFIG file, it will fallback to credential manager. If it doesn't find the credentials in KEPTNCONFIG, it won't throw any error, it will return `""` apitoken and hence for that it should again fallback to credential manager.

Reply:
I can also put the check like `if err != nil && errr.Error() != "Not found"` by sending custom error.

Reply:
OK, got it. But still, if `handleCustomCreds` returns an empty `apiToken` and an `error`, the `GetCreds()` function would return that error instead of falling back to the [credential manager](https://github.com/keptn/keptn/blob/0d0b71de7f44dcb7e1387384d13a925df913b413/cli/pkg/credentialmanager/handler_linux.go#L73) - is this intended?

Reply:
Yes, This is intended, since it's an error not the case of credentials not found so the end-user should be aware of the actual error.

Reply:
ok that makes sense. thank you for the explanation. In this case, I think we can merge the PR